service/service.go

```diff
@@ -195,18 +211,30 @@ func (s *Service) BackupGuestClusters() error {
 
 			TmpDir: tmpDir,
 		}
-		// run backup task
-		err = etcd.FullBackup(&backupConfig)
+
+		o := func() error {
+
+			err = etcd.FullBackup(&backupConfig)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			s.Logger.Log("level", "info", "msg", "Cluster backup created for: "+clusterID)
+			return nil
+		}
+
+		b := backoff.NewMaxRetries(10, 20*time.Second)
```

Comment:
Let's please use retries here on literal in the previous `backoff.NewMaxRetries` call.

Reply:
Typo. Good catch